gitea/repo_collaborator.go

```diff
@@ -10,6 +10,29 @@ import (
 	"fmt"
 )
 
+// ListCollaborators list a repository's collaborators
+func (c *Client) ListCollaborators(user, repo string) ([]*User, error) {
```

Comment:
@lunny @bkcsoft @tboerger Do we want pager or not? Happy to add one if we do, just want to make sure.